.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
```

Comment:
hmm... we already have a flake8 block in setup.cfg... could we keep editing it there?

Reply:
i don't mean to do a hard block on this. if it's too big of a pain in the back to fix this, don't worry and i'll remove by 'request changes' ;) i just figured you might have missed the entries in setup.cfg that @mgedmin added recently

Reply:
Indeed I did (I have had this for a while and published it without checking), so good block.

Reply:
Moved to setup.cfg.